packages/screens/MusicPlayer/MusicPlayerScreen.tsx

```diff
@@ -1,25 +1,54 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { View, StyleSheet } from "react-native";
 
 import { MusicPlayerHomeContent } from "./MusicPlayerHomeContent";
 import { MusicPlayerMyLibraryContent } from "./MusicPlayerMyLibraryContent";
+import { GetAllAlbumListRequest } from "../../api/musicplayer/v1/musicplayer";
 import { BrandText } from "../../components/BrandText";
 import { MediaPlayer } from "../../components/MusicPlayer/MediaPlayer";
 import { MusicPlayerTab } from "../../components/MusicPlayer/MusicPlayerTab";
 import { ScreenContainer } from "../../components/ScreenContainer";
+import { useSelectedNetworkId } from "../../hooks/useSelectedNetwork";
+import useSelectedWallet from "../../hooks/useSelectedWallet";
+import { getUserId } from "../../networks";
+import { mustGetMusicplayerClient } from "../../utils/backend";
 import { ScreenFC } from "../../utils/navigation";
 import { neutralA3, secondaryColor } from "../../utils/style/colors";
 import { fontSemibold14 } from "../../utils/style/fonts";
 import { layout } from "../../utils/style/layout";
-import { GetAllAlbumListRequest } from "../../api/musicplayer/v1/musicplayer";
+
 export const MusicPlayerScreen: ScreenFC<"MusicPlayer"> = () => {
   const tabData: string[] = ["Home", "My Library"];
   const [tab, setTab] = useState<string>(tabData[0]);
+  const [idForLibraryList, setIdForLibraryList] = useState<string[]>([]);
+  const selectedNetworkId = useSelectedNetworkId();
+  const wallet = useSelectedWallet();
+  const userId = getUserId(selectedNetworkId, wallet?.address);
 
   const musicRequest: GetAllAlbumListRequest = {
-      limit: 10,
-      offset: 0,
+    limit: 10,
+    offset: 0,
   };
+  useEffect(() => {
+    const getLibraryIdList = async () => {
+      if (!userId) return;
+      try {
+        const res = await mustGetMusicplayerClient(
+          selectedNetworkId
+        ).GetAlbumIdListForLibrary({
+          user: userId,
+        });
+        const idList: string[] = [];
+        res.albumLibraries.map((libraryInfo, index) => {
+          idList.push(libraryInfo.identifier);
+        });
+        setIdForLibraryList(idList);
+      } catch (err) {
+        console.log(err);
+      }
+    };
+    getLibraryIdList();
+  }, [selectedNetworkId, userId]);
```

Comment:
please use react-query for fetching data

Reply:
done